app/controllers/v0/sessions_controller.rb

```diff
@@ -99,17 +100,20 @@ def user_login(saml_response)
       if user_session_form.valid?
         @current_user, @session_object = user_session_form.persist
         set_cookies
-        # track users who need to re-login on MHV
-        StatsD.increment(STATSD_MHV_COOKIE_NO_ACCOUNT_KEY) unless @current_user.mhv_correlation_id
         after_login_actions
         redirect_to url_service.login_redirect_url
-        stats(:success, saml_response)
+        if location.start_with?(url_service.base_redirect_url)
```

Comment:
is there a corresponding PR for the alert rules?  https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/prometheus/rules/external_service.rules.j2#L100 . If this goes out without a reduction in the alert threshold, someone will get paged and users will see a downtime notification that looks like this https://github.com/department-of-veterans-affairs/devops/blob/master/docs/External%20Service%20Integrations/DS_logon_Banner.png

Reply:
the existing `stats` method was renamed to `callback_stats`.  These are the values that are added on every callback, and should be what the alerts are triggered off of.  The new `login_stats` method, only gets called once per user login, and is tracking new metrics that are not being used in any alert rules. https://github.com/department-of-veterans-affairs/vets-api/pull/3762#issuecomment-583475348